src/text-buffer.js

```diff
@@ -2601,4 +2614,18 @@ class SearchCallbackArgument {
   }
 }
 
+let _winattr = null
```

Comment:
Do you want it to live somewhere else in the file? Maybe at the top? I added it there because in order to understand the `getPromisifiedWinattr` function you can just look at that piece of code and understand it all, no need to scroll to know what `_winattr` is. But I'm happy to move it somewhere else to follow Atom's conventions.

Reply:
I wasn't getting the idea behind defining it there. But I think I see the point now. Correct me if am work. It is defined here so that calling `getPromisifiedWinattr` multiple times does not result in multiple `require` calls.

Reply:
Yeah that's right. It will also only promisify the methods once.